ocpmodels/common/utils.py

```diff
@@ -211,7 +211,7 @@ def setup_imports():
     datasets_folder = os.path.join(root_folder, "datasets")
     datasets_pattern = os.path.join(datasets_folder, "**", "*.py")
     model_folder = os.path.join(root_folder, "models")
-    model_pattern = os.path.join(model_folder, "**", "*.py")
+    model_pattern = os.path.join(model_folder, "*.py")
```

Comment:
@abhshkdz @sidgoyal78 

Per how these imports were setup, if this change wasn't made I would hit this error:
```
Traceback (most recent call last):
  File "main.py", line 115, in <module>
    main(config)
  File "main.py", line 30, in main
    setup_imports()
  File "/home/jovyan/projects/ocp/ocpmodels/common/utils.py", line 231, in setup_imports
    "ocpmodels.%s.%s" % (key[1:], module_name)
  File "/home/jovyan/miniconda3/envs/ocp-models/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'ocpmodels.models.atomic_radii'
```

This holds true for any subdirectory created within `models`, `datasets`, and `trainers`. Let me know if we have a better way to dealing with this.

Reply:
We might need a __init__.py file in `models/utils/` as well. Could you check if that helps?
I see that you have a __init__.py in `models/utils/embeddings/` so this would be at one dir above.

Reply:
This didn't work either.